src/api/server/server.js

```diff
@@ -37,19 +69,19 @@ function getLedgerVersion(): number {
   return this.remote.getLedgerSequence();
 }
 
-function connect() {
+function connect(): Promise {
   return common.promisify(callback => {
     this.remote.connect(() => callback(null));
   })();
 }
 
-function disconnect() {
+function disconnect(): Promise {
   return common.promisify(callback => {
     this.remote.disconnect(() => callback(null));
   })();
 }
 
-function getServerInfo() {
+function getServerInfo(): Promise {
```

Comment:
I've didn't find it too.
But now, finally, by looking into flow's code I've found how to annotate it - it is just `Promise<Type>`
I'll make changes to PR